### ISSUE TITLE
Wrong path for array type parameters in OAS3

### ISSUE THREAD

**Issue by kannkyo** [apiaryio/dredd#1737]
**Describe the bug**

My test result was failure because of wrong path`/pet/findByStatus?status=available,pending,sold`.

```
Fail: GET (200) /pet/findByStatus?status=available,pending,sold
Message
headers: At '/content-type' No enum match for: "application/json; charset=utf-8"
body: At '' Invalid type: object (expected array)
statusCode: Expected status code '200', but got '400'.
```

**To Reproduce**

When using OAS3 file like bellow, I got the failure.

openapi.yaml

```yaml
paths:
  /pet/findByStatus:
    get:
      tags:
        - pet
      summary: Finds Pets by status
      description: Multiple status values can be provided with comma separated strings
      operationId: findPetsByStatus
      parameters:
        - name: status
          in: query
          description: Status values that need to be considered for filter
          required: true
          style: form
          explode: true
          schema:
            type: array
            items:
              type: string
              default: available
              enum:
                - available
                - pending
                - sold
          example: [available, pending, sold]
```

**Expected behavior**

1. Expected path : `/pet/findByStatus?status=available&status=pending&status=sold`
2. Actual path: `/pet/findByStatus?status=available,pending,sold`

Swagger-codegen (Swagger official code generator) generate 2. actual path as node.js server.
So, I guess that 2. is correct path for OAS3.

**What is in your `dredd.yml`?**

```yaml
color: true
dry-run: null
hookfiles: null
language: nodejs
require: null
server: npm start
server-wait: 3
init: false
custom: {}
names: false
only: []
reporter: [markdown, html]
output: []
header: ["api_key: something"]
sorted: false
user: null
inline-errors: false
details: false
method: []
loglevel: warning
path: []
hooks-worker-timeout: 5000
hooks-worker-connect-timeout: 1500
hooks-worker-connect-retry: 500
hooks-worker-after-connect-wait: 100
hooks-worker-term-timeout: 5000
hooks-worker-term-retry: 500
hooks-worker-handler-host: 127.0.0.1
hooks-worker-handler-port: 61321
config: ./dredd.yml
blueprint: ./api/openapi.yaml
endpoint: "http://localhost:2080/v2/"
```

**What's your `dredd --version` output?**

```
dredd v13.1.0 (Linux 4.15.0-96-generic; x64)
```

**Does `dredd --loglevel=debug` uncover something?**

No.

**Can you send us failing test in a Pull Request?**

No.

**Comment by kylef** [comment]
Hi @kannkyo, I've published dredd@13.1.1 which adds support for the OpenAPI 3 "explode" modifier for query parameters. Can you please give this a go.

**Comment by kannkyo** [comment]
Hi @kylef,it's looks good to me.
I can see "Pass" at my report.

```
Pass: GET (200) /pet/findByStatus?status=available&status=pending&status=sold
```

Thank you.
